app/controllers/users_controller.rb

```diff
@@ -1,6 +1,14 @@
 class UsersController < ApplicationController
+  before_action :logged_in_user, expect: %i(:show :new :create)
+  before_action :correct_user, only: %i(:edit :update)
+  before_action :admin_user, only: %i(:destroy)
+  before_action :load_user, expect: %i(:index :new :create)
+
+  def index
+    @users = User.order(:name).page params[:page]
```

Comment:
hiện tại trang mặc định là bao nhiêu bản ghi, anh muốn chuyển thành 10 bản ghi 1 trang thì làm như nào

Reply:
Dạ, trang mặc định là 25 bản ghi ạ.
- Để sửa thành 10 bản ghi thì thêm hàm built-in per(10) vào là được ạ. 
@users = User.order(:name).page params[:page].per(10)

Reply:
sửa thành 10 bản ghi 1 trang, 10 --> Settings